homeassistant/components/eufylife_ble/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for the EufyLife integration."""
+
+DOMAIN = "eufylife_ble"
+
+CONF_MODEL = "model"
+
+EUFYLIFE_MODEL_TO_NAME = {
+    "eufy T9140": "Smart Scale",
+    "eufy T9146": "Smart Scale C1",
+    "eufy T9147": "Smart Scale P1",
+    "eufy T9148": "Smart Scale P2",
+    "eufy T9149": "Smart Scale P2 Pro",
+}
```

Comment:
Looks like nothing in this is HA specific so it could live in the lib

Reply:
Fixed.